### DISCUSSION TITLE
I cant get a Templater file to render when placed

### DISCUSSION THREAD

**Discussion by dr34mc0d3r** [SilentVoid13/Templater#1448] [Help]
I have a fresh Obsidian Vault and only Templater plugin installed. I created a new folder and called it Templates. I configured Templater to point to the new Templates folder "Template folder location".

In Templates folder i created a new note called "Template1". I see in the vault that the file is called "Template1.md".
I edit Template1 with a very basic command:

`<%*
tp.file.last_modified_date();
%>
`

Then I create a new note. CMD-P and search for Template and select Insert Template1. I see the command inserted. I switch to Reading view and i see the code. 

I CMD-Z to undo. Then I try CMD-P search for Template and select "Replace templates in active file". I see the desired output.
I wait a few minutes and make a modification. Then close the note
.
But when I close the note and then re-open it - i still see the modified time as earlier. It appears that the command didnt run again even though the Template is applied to the note.

What am i missing?

**Comment by Zachatoo** [comment]
Templater is not a good option for content that dynamically changes on render, see #913 and comments for various alternatives and how to use them.

I would suggest Dataview or Linter for this use case.
